Software_Carbon_Intensity/Appendix_A_Further_Information_on_Reporting_Requirements.md

```diff
@@ -0,0 +1,98 @@
+# Appendix A: Further information on Reporting Requirements
+This appendix provides additional detail on validly formed machine-readable SCI reporting files, in YAML format.
+
+## File Structure
+
+To comply with this specification and implement carbon transparency, an entity MUST meet these reporting requirements. The hierachical structure of the YAML file is described below.
+
+### Data elements to be reported
+The following list contains the REQUIRED and OPTIONAL data elements to be reported.
+
+| Name | Identifier | Optionality | Format | Notes |
+| - | - | - | - | - |
+| Supporting infrastructure and systems contained within the [Software Boundary](#software-boundary) | sci/software-boundary | MUST | Text | A description of the software boundary for the entity. |
+| [Software Carbon Intensity](#reporting-the-sci-value) - `CI`| sci/ci | MUST | Numeric | The [Software Carbon Intensity](#reporting-the-sci-value) of the entity itself.
```

Comment:
The markdown URLs here need to be updated - left in until we've confirmed section names. Changes to be made prior to call on Thursday